### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by vfonic**
If this PR is too difficult to merge at this point, I can break it down into several smaller parts.

I'm not sure if some of these are breaking changes and/or if some of these are better left without being fixed.

### CODE REVIEW

File: api/lib/spree/api/testing_support/v2/platform_contexts.rb [hunk 1]
```diff
@@ -50,12 +50,12 @@ def articleize
...
 def json_api_include_parameter(example = '')
   let(:include) { nil }
-  parameter name: :include, in: :query, type: :string, descripton: JSON_API_INCLUDES_DESCRIPTION, example: example
+  parameter name: :include, in: :query, type: :string, description: JSON_API_INCLUDES_DESCRIPTION, example: example
 end
 
 def json_api_filter_parameter(example = '')
   let(:filter) { nil }
-  parameter name: :filter, in: :query, type: :string, descripton: JSON_API_FILTER_DESCRIPTION, example: example
+  parameter name: :filter, in: :query, type: :string, description: JSON_API_FILTER_DESCRIPTION, example: example
```
Comment (vfonic) [inline api/lib/spree/api/testing_support/v2/platform_contexts.rb:58]: I'm not sure if these two changes somehow might make the tests fail.

File: core/app/models/spree/store.rb [hunk 2]
```diff
@@ -76,8 +76,8 @@ class Store < Spree::Base
 
     def self.current(url = nil)
       ActiveSupport::Deprecation.warn(<<-DEPRECATION, caller)
-        `Spree::Store.current` is deprecated and will be removed in Spree 5
-        Spree::Stores::FindCurrent.new(url: "http://example.com") instead
+        `Spree::Store.current` is deprecated and will be removed in Spree 5.0
+        Please use `Spree::Stores::FindCurrent.new(url: "https://example.com").execute` instead
```
Comment (vfonic) [inline core/app/models/spree/store.rb:80]: I've aligned this deprecation warning with the other deprecation warnings. It didn't seem like:
`Spree::Stores::FindCurrent.new(url: "http://example.com") instead` is a complete sentence

File: core/config/locales/en.yml [hunk 3]
```diff
@@ -1138,7 +1138,7 @@ en:
     log_in_to_continue: Log in to continue
     logs: "Logs"
     logged_in_as: Logged in as
-    logged_in_succesfully: Logged in successfully
+    logged_in_successfully: Logged in successfully
```
Comment (vfonic) [inline core/config/locales/en.yml:1141]: I'm not sure if this will break anything...

**Review by damianlegawiec — APPROVED** [review]

File: core/config/locales/en.yml (hunk 3 above)
Comment (damianlegawiec) [reply core/config/locales/en.yml:1141]: @vfonic could probably break translations in spree_i18n but that's an easy fix

File: api/lib/spree/api/testing_support/v2/platform_contexts.rb (hunk 1 above)
Comment (damianlegawiec) [reply api/lib/spree/api/testing_support/v2/platform_contexts.rb:58]: the description is an optional parameter so no worries, it didn't fail with the typo ;)

**Pull request merged**